code/modules/mob/living/simple_animal/hostile/mining_mobs/basilisk.dm

```diff
@@ -98,14 +97,16 @@
 	icon_living = "watcher_magmawing"
 	icon_aggro = "watcher_magmawing"
 	icon_dead = "watcher_magmawing_dead"
-	maxHealth = 215 //Compensate for the lack of slowdown on projectiles with a bit of extra health
-	health = 215
+	maxHealth = 235 //Compensate for the lack of slowdown on projectiles with a bit of extra health
+	health = 235
 	light_range = 3
 	light_power = 2.5
 	light_color = LIGHT_COLOR_LAVA
 	projectiletype = /obj/item/projectile/temp/basilisk/magmawing
 	crusher_loot = /obj/item/crusher_trophy/blaster_tubes/magma_wing
-	crusher_drop_mod = 60
+	crusher_drop_mod = 10
+	loot = list()
+	butcher_results = list(/obj/item/stack/ore/diamond = 3, /obj/item/stack/sheet/sinew = 2, /obj/item/stack/sheet/bone = 1)
```

Comment:
I don't really understand why they get more loot when they're easier to fight?

Reply:
Is it not the higher the number the higher the drop chance.

Reply:
also i buffed them to make them a bit more difficult and sense they are "more rare" they have more loot